### DISCUSSION TITLE
Status Page

### DISCUSSION THREAD

**Discussion by NuroDev** [planetscale/discussion#47] [Feature Requests] [answered]
Not a high priority, but it would nice to have some kind of status page, something like [status.planetscale.com](https://status.planetscale.com), to show the current platform status, any outstanding issues/incidents, etc.

**Comment by NuroDev** [comment, marked as answer]
Never mind, I have just found [planetscale.freshstatus.io](https://planetscale.freshstatus.io/).
Would be nice if that could get aliased to [status.planetscale.com](https://status.planetscale.com) for simplicity reasons?

**Reply by hallaroo** [reply]
Hi @NuroDev! You are indeed correct that we have [planetscale.freshstatus.io](planetscale.freshstatus.io). I'm checking in with the team to see what we want to do in regards to setting up an alias or custom domain URL. 

I'll keep you posted on what I hear back. Thank you so very much for the suggestion.
